e2e/tests/regression/test_backup.robot

```diff
@@ -44,3 +49,58 @@ Test Backup Volume List
 
     And Delete backup volume 0
     And Delete backup volume 1
+
+Test Incremental Restore
+    [Documentation]    Test restore from disaster recovery volume (incremental restore)
+    Given Create volume 0
+    And Attach volume 0
+    And Wait for volume 0 healthy
+    And Write data 0 to volume 0
+    And Create backup 0 for volume 0
+
+    When Create DR volume 1 from backup 0
+    And Wait for volume 1 restoration from backup 0 completed
+    And Create DR volume 2 from backup 0
+    And Wait for volume 2 restoration from backup 0 completed
+    And Create DR volume 3 from backup 0
+    And Wait for volume 3 restoration from backup 0 completed
+
+    # Verify snapshot creation and PV/PVC creation are not allowed on a DR volume
+    Then Create snapshot 0 of volume 1 will fail
+    And Create persistentvolume for volume 1 will fail
+    And Create persistentvolumeclaim for volume 1 will fail
```

Comment:
One keyword that might help us better understand the story in the test report.
```
*** Keywords ***
Snapshot PV PVC could not be created on DR volume 1
    Create snapshot 0 of volume 1 will fail
    Create persistentvolume for volume 1 will fail
    Create persistentvolumeclaim for volume 1 will fail

Backup target could not be changed when DR volume exist
    Set setting backup-target to random.backup.target will fail


*** Test Cases ***
...
    And Wait for volume 3 restoration from backup 0 completed
    
    Then Snapshot PV PVC could not be created on DR volume 1
    And Backup target could not be changed when DR volume exist

    When Activate DR volume 1
...
```

![Screenshot_20240607_161752](https://github.com/longhorn/longhorn-tests/assets/104337648/5401b061-3af7-4905-ba2f-e41866398779)

Reply:
Refined. Thank you!